packages/block-editor/src/style.scss

```diff
@@ -36,6 +36,7 @@
 @import "./components/media-placeholder/style.scss";
 @import "./components/multi-selection-inspector/style.scss";
 @import "./components/plain-text/style.scss";
+@import "./components/preview-options/style.scss";
```

Comment:
This should be moved to beneath line 55, under the marker for resizable styles, so that its contents aren't impacted by the device preview style manipulation.

Reply:
Updated in dcbaf85d0da3b2d90cfce7110cd19e8c925fad3b.